modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/component/IconCountBadge.kt

```diff
@@ -0,0 +1,16 @@
+package au.com.shiftyjelly.pocketcasts.views.component
+
+import android.content.Context
+import au.com.shiftyjelly.pocketcasts.ui.R
+import au.com.shiftyjelly.pocketcasts.ui.extensions.getThemeColor
+import au.com.shiftyjelly.pocketcasts.utils.extensions.dpToPx
+import com.google.android.material.badge.BadgeDrawable
+
+fun createIconCountBadge(context: Context) =
```

Comment:
It might be but I'm having a little trouble with wording. Do we count icons on this badge?

Reply:
How does just `createCountBadge` sound?

Reply:
Yup, thank you 🙏

Reply:
Renamed in [9a0dbaa](https://github.com/Automattic/pocket-casts-android/pull/2746/commits/9a0dbaa9211d6dc381b74a868d10feea88aa731e)